.github/workflows/test.yml

```diff
@@ -78,7 +78,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-            node-version: 12
+            node-version: 17.9.0
```

Comment:
I think we should go with the LTS version 16. Node version also needs to be upgraded in adx_deploy / adx_develop (it's installed in the ckan/Dockerfile).